invenio_rdm_records/contrib/meeting/custom_fields.py

```diff
@@ -18,12 +18,37 @@
 
 from invenio_i18n import lazy_gettext as _
 from invenio_records_resources.services.custom_fields import BaseCF
-from marshmallow import fields, validate
+from marshmallow import fields
 from marshmallow_utils.fields import SanitizedUnicode
 
+from invenio_rdm_records.contrib.journal.custom_fields import FieldDumper
 from invenio_rdm_records.services.schemas.metadata import _valid_url
 
 
+class MeetingDublinCoreDumper(FieldDumper):
+    """Dump processor for dublin core serialization of 'Meeting' custom field."""
+
+    def post_dump(self, data, original={}, **kwargs):
```

Comment:
```suggestion
    def post_dump(self, data, original=None, **kwargs):
```

Reply:
This needs to be fixed :)
and add below:
```
_original = original or {}
```